paparazzi-gradle-plugin/src/test/projects/custom-report-dir/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+  id 'com.android.library'
+  id 'kotlin-android'
+  id 'app.cash.paparazzi'
+}
+
+reporting {
+  baseDir 'custom/our-reports'
+}
```

Comment:
nit: any interest in making this block `paparazzi { reporting { baseDir '...' } }`. I could see this possibly being leveraged to enable validate accessibility errors and configure allow lists in the future

Reply:
You could make a `paparazzi { reportDir .. }`, but that property within your own extension would use a default value sourced from the `reporting { }` `baseDir` value as it's a standard Gradle mechanism.

Reply:
I think to take a step back: the question is if it's necessary to allow users of Paparazzi to specifically move the reports folder, or it's enough to put Paparazzi reports in the same places as "all the other reports". I think it would be nice to be able to do it, but potentially it wouldn't be used as much.

Reply:
@geoff-powell As the rest have mentioned, this is more of a general reporting configuration vs a Paparazzi-specific one.